composer.json

```diff
@@ -59,7 +59,8 @@
         "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
         "oat-sa/lib-lti1p3-core": "^6.0.0",
         "oat-sa/generis" : ">=14.0.0",
-        "oat-sa/tao-core" : ">=48.30.0"
+        "oat-sa/tao-core" : ">=48.30.0",
+        "oat-sa/lib-lti1p3-ags": "^1.2"
```

Comment:
Can we keep it sorted alphabetically, please?

You can either set it in the config `"sort-packages": true` or provide the option `--sort-packages` while running the composer command.

It's very helpful when reading the dependencies.